index.html

```diff
@@ -244,53 +286,58 @@ <h3 class="skill-img ab"> <img src="img/framwork56.png" alt=""> Framework</h3>
         </div>
```

Comment:
- [ Optional ] Good job for trying to fix the overflow problem the last reviewer mentioned but please add the over-flow hidden to the `*` is not the solution you are adding an overflow for each element in the html doc instead you can add this property to the html and body tags 

![image](https://user-images.githubusercontent.com/74744787/215293728-8e98dc17-c8d3-4bf1-936a-e890670fddbb.png)